tests/sentry/api/test_event_search.py

```diff
@@ -398,3 +398,22 @@ def test_is_query_unsupported(self):
             InvalidSearchQuery, ".*queries are not supported in this search.*"
         ):
             parse_search_query("is:unassigned")
+
+    def test_escaping_asterisk(self):
```

Comment:
Could we add an xfail for the edgecase where a string ends with a `\`?

Reply:
added in 0f0c5ff